README.md

```diff
@@ -135,6 +135,7 @@ Pikaday has many useful options:
 * `firstDay` first day of the week (0: Sunday, 1: Monday, etc)
 * `minDate` the minimum/earliest date that can be selected (this should be a native Date object - e.g. `new Date()` or `moment().toDate()`)
 * `maxDate` the maximum/latest date that can be selected (this should be a native Date object - e.g. `new Date()` or `moment().toDate()`)
+* `yearOrderMinMax` the sorted order of the years displayed in the dropdown, defaults to true
```

Comment:
I would recommend option name `yearOrder` nad value `ascending` / `descending`.

Reply:
Please refer to [pull request 833](https://github.com/Pikaday/Pikaday/pull/833).